src/components-examples/material/table/index.ts

```diff
@@ -46,39 +46,43 @@ import {TableHarnessExample} from './table-harness/table-harness-example';
 import {TableWithRipplesExample} from './table-with-ripples/table-with-ripples-example';
 import {TableColumnStylingExample} from './table-column-styling/table-column-styling-example';
 import {TableRowBindingExample} from './table-row-binding/table-row-binding-example';
+import {TableDynamicArrayDataExample} from './table-dynamic-array-data/table-dynamic-array-data-example';
+import {TableDynamicObservableDataExample} from './table-dynamic-observable-data/table-dynamic-observable-data-example';
```

Comment:
Just writing this here: Lint fails because these two imports exceed 100 columns.